osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -126,7 +126,7 @@ private void load(FrameworkConfigManager config, OsuGameBase game)
 
         private List<KeyValuePair<string, int>> getResolutions()
         {
-            var availableDisplayResolutions = (game.Window as DesktopGameWindow)?.AvailableDisplayResolutions
+            var availableDisplayResolutions = (game.Window as DesktopGameWindow)?.GetCurrentDisplay().AvailableResolutions
```

Comment:
`GetCurrentDisplay` is a member of `GameWindow` on purpose - no need to cast :)
Also, you should call `.Distinct()` here (it got lost with the `AvailableDisplayResolutions` property)